src/EPoetryClientFactory.php

```diff
@@ -38,6 +41,12 @@ class EPoetryClientFactory
      */
     protected $logger;
 
+    /**
+     * Level of severity of the events to be logged.
+     * @var bool | string
+     */
+    protected $logLevel = false;
```

Comment:
Can we maybe have a value like "none" to specify that we don't want any logs? So that this property can just be a string.